docs/controls/BubbleMultipleDataset.md

```diff
@@ -0,0 +1,274 @@
+    }
+    values: [
+        {
+            x: "2016-03-01T12:00:00Z",
+            y: 14,
+        },
+        {
+            x: "2016-04-10T12:00:00Z",
+            y: 1,
+        },
+        {
+            x: "2016-11-01T12:00:00Z",
+            y: 18,
+        }
+    ]
+};
+bubbleGraph = Carbon.api.graph(root);
+bubbleGraph.loadContent(Carbon.api.bubbleMultipleDataset(customSizeBasedData));
+```
+
+## JSON Properties
+
+### Root
+
+Refer [Graph](../core/Graph.md) `Root` for more details.
+
+### Data
+
+#### Required
+
+| Property Name | Expected | Description                             |
+| ------------- | -------- | --------------------------------------- |
+| key           | string   | Unique id which represents the data-set |
+| values        | Array    | [Values](#values)                       |
+
+#### Optional
+
+| Property Name | Expected | Default      | Description                                                                               |
+| ------------- | -------- | ------------ | ----------------------------------------------------------------------------------------- |
+| color         | string   | COLORS.BLACK | Color for the bubbles                                                                     |
+| onClick       | Function | undefined    | Any action that can be performed when clicking on the data point                          |
+| hue           | object   | {}           | Provide lowerShade and upperShadefor color based bubble [Hue](#Hue)                       |
+| label         | object   | {}           | Display value for the data-set which the data points belong to                            |
+| legendOptions | object   | undefined    | Toggle to show legend. Refer [LegendOptions](#legendOptions)                              |
+| regions       | array    | []           | Refer [Regions](#regions)                                                                 |
+| weight        | object   | {}           | Provide min and max or maxRadius for weight based or custom sized bubble[Weight](#Weight) |
+| yAxis         | string   | "y"          | Setting for using different Y based axis. For now: its either Y or Y2                     |
+
+### LegendOptions
+
+Each bubble can have a legendOptions object in [Values](#values) level.
+
+#### Optional
+
+| Property Name | Expected | Default | Description                                         |
+| ------------- | -------- | ------- | --------------------------------------------------- |
+| showElement   | boolean  | true    | Toggle to hide legend when legend item has no data. |
+
+### Weight
+
+For weight based bubble provide weight with `min` and `max`, each bubble can have a weight object in [Values](#values) level.
+
+```
+weight: {
+    min: 10,
+    max: 30,
+    maxRadius: 30
+}
+```
+
+#### Required
+
+| Property Name | Expected | Default   | Description                        |
+| ------------- | -------- | --------- | ---------------------------------- |
```

Comment:
Do we still need lower shade and upper shade properties?

Reply:
Removed: 
https://github.com/cerner/carbon-graphs/pull/308/commits/1a920bb9dc6643049ba404be1a3bcc8cd672e8ac